### DISCUSSION TITLE
How to using query iterators with new MilvusClient method?

### DISCUSSION THREAD

**Discussion by secsilm** [milvus-io/milvus#33143] [Q&A and General discussion]
The current [document](https://milvus.io/docs/with-iterators.md#Query-with-an-iterator) uses the old way.

**Comment by yhmo** [comment]
MilvusClient has no interface for iterator. Just use the old way.

**Comment by xiaofan-luan** [comment]
@czs007 let's add one.
iter:=MilvusClient.open
defer iter.Close()
while iter.hasNext()
   process(iter.next())

**Reply by czs007** [reply]
@secsilm @xiaofan-luan  
LGTM 
It's on my schedule.
